cluster-scope/components/project-admin-rolebindings/peribolos/rbac.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
This wont be required, we wont need any cluster role or cluster rolebinding , TIL i know

Reply:
this is a regular rolebinding, it just gives namespace admin to the namespace being created, for the group also being created

Reply:
ah right, thanks for clarification.
no changes needed then.